lib/pages/chat/events/message.dart

```diff
@@ -57,527 +70,539 @@ class Message extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return LayoutBuilder(
-      builder: (context, constraints) {
-        if (!{
-          EventTypes.Message,
-          EventTypes.Sticker,
-          EventTypes.Encrypted,
-          EventTypes.CallInvite
-        }.contains(event.type)) {
-          if (event.type.startsWith('m.call.')) {
-            return Container();
+    return InkWell(
+      hoverColor: Colors.transparent,
+      onTap: () {},
```

Comment:
do nothing ?

Reply:
Yes, I just want hover. 
Why I define `onTap` becase this behavior is actually as intended. When onTap is not defined, the widget is disabled, and disabled widgets aren't supposed to respond to hover. If you wish to enable the widget, but do nothing when tapped, then do what you did in the example above: define an empty callback for onTap.